src/deserialize_sequence.cc

```diff
@@ -319,4 +321,23 @@ void load_sequence_parameters(const std::filesystem::path& folder, Sequence& seq
     sequence.set_step_setup_script(step_setup_script);
 }
 
+std::vector<Tag> parse_tags(gul14::string_view str)
+{
+    const auto tokens = gul14::tokenize_sv(str);
```

Comment:
Hmm, this means we ignore some invalid characters like `\t` but not other like `_`.

I would enforce 'only blank is allowed' via

```suggestion
    const auto tokens = gul14::tokenize_sv(str, " ");
```

Reply:
I do not see the advantage in that. We still theoretically allow these .lua files to be edited by hand, and some people are fond of tabs. So why prohibit them?

Reply:
Fair enough ;)